### PR TITLE
Add data guideline regarding constants

### PR DESCRIPTION
**Pull request by sideshowbarker**
See https://github.com/mdn/browser-compat-data/issues/7936#issuecomment-747390760 and https://github.com/mdn/browser-compat-data/issues/7585#issuecomment-739907251

### CODE REVIEW

**Comment by foolip** [comment]
I spotted some constants in https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent, are there others that will need cleaning up? If any of them have notes we should read them before deciding the entries have to go :)

File: docs/data-guidelines.md [hunk 1]
```diff
@@ -177,6 +177,12 @@ However, this guideline does not apply to features where the browser's expected
 
 This guideline was proposed in [#6906](https://github.com/mdn/browser-compat-data/issues/6906).
 
+## Constants
+
+We don't include data for constants in BCD. The rationale for not including them is that they're not known to be a source of any compatibility issues.
```
Comment (ddbeck) [inline docs/data-guidelines.md:182]:
```suggestion
Don't include data for constants in BCD. The rationale for not including them is that they're not known to be a source of any compatibility issues.
```

File: docs/data-guidelines.md [hunk 2]
```diff
@@ -177,6 +177,12 @@ However, this guideline does not apply to features where the browser's expected
...
+
```
Comment (ddbeck) [inline docs/data-guidelines.md:183]: It'd be nice to have a concrete example of a constant that we would omit from BCD.

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thank you for starting this, @sideshowbarker!

File: docs/data-guidelines.md [hunk 3]
```diff
@@ -177,6 +177,12 @@ However, this guideline does not apply to features where the browser's expected
 
 This guideline was proposed in [#6906](https://github.com/mdn/browser-compat-data/issues/6906).
 
+## Constants
```
Comment (ddbeck) [inline docs/data-guidelines.md:180]: Oh, I missed one thing: can you update the ToC at the top of this file? Line 5 has instructions for doing this easily-ish

Comment (sideshowbarker) [reply docs/data-guidelines.md:180]: Will do

File: docs/data-guidelines.md (hunk 2 above)
Comment (sideshowbarker) [reply docs/data-guidelines.md:183]: Agreed. Will add

Comment (sideshowbarker) [reply docs/data-guidelines.md:185]: > It'd be nice to have a concrete example of a constant that we would omit from BCD.

OK, added

File: docs/data-guidelines.md (hunk 3 above)
Comment (sideshowbarker) [reply docs/data-guidelines.md:181]: > Oh, I missed one thing: can you update the ToC at the top of this file? Line 5 has instructions for doing this easily-ish

OK, ToC updated

**Review by ddbeck — APPROVED** [review]
OK, I'm happy with this and I'm going to merge it. Note that this only rejects adding new constants data to BCD. We'll need a separate issue for finding and pruning constants that have already slipped in, if that's something we want to do.

Thank you, @sideshowbarker!

**Pull request merged**